### ISSUE TITLE
Small typo in console command line switch

### ISSUE THREAD

**Issue by sanderobdeijn** [xBimTeam/Xbim.IDS.Validator#37]
In https://github.com/xBimTeam/Xbim.IDS.Validator/blob/master/Xbim.IDS.Validator.Console/CliOptions.cs
the verbosity option is set as --vebosity instead of --verbosity
`new Option<Verbosity> (aliases: new[] { "--vebosity", "-v" }, description: "Verbosity of the output")`

And in https://github.com/xBimTeam/Xbim.IDS.Validator/blob/master/Xbim.IDS.Validator.Console/Commands/IdsDetokeniseCommand.cs there seams to be an copy paste error in class comment from the IdAuditCommand.
```
    /// <summary>
    /// Audits IDS files against the IDS 1.0 standards using ids-lib
    /// </summary>
    internal class IdsDetokeniseCommand : ICommand
```

**Comment by andyward** [comment]
Thanks for reporting @sanderobdeijn - will get this fixed!

**Comment by andyward** [comment]
Fixed in 76fd70688fd569ec89eebb9a3732c8f2263b0221
